docs/usage.md

```diff
@@ -1,5 +1,35 @@
 # Usage
 
+## tach mod
+`tach` comes bundled with a command to set up and define your initial boundaries.
+
+```bash
+usage: tach mod [-h] [-d [DEPTH]] [-e file_or_path,...]
+
+Configure module boundaries interactively
+
+options:
+  -h, --help            show this help message and exit
+  -d [DEPTH], --depth [DEPTH]
+                        The number of child directories to expand from the root
+  -e file_or_path,..., --exclude file_or_path,...
+                        Comma separated path list to exclude. tests/, ci/, etc.
+```
+
+Running `tach mod` will open an interactive editor in your terminal which allows you to mark your module boundaries.
+
+You can navigate with the arrow keys, mark individual modules with `Enter`, and mark all siblings
+as modules with `Ctrl + a`.
+
+You can also mark your Python [source root](configuration.md#source-root) by pressing `s`.
+This allows Tach to understand the module paths used in your project, and to correctly identify first-party imports.
+
+When you are ready to save your modules, use `Ctrl + s` to save and exit. Otherwise, to exit without saving you can use `Ctrl + c`.
+
+Any time you make changes with `tach mod`, it is recommended to run [`tach sync`](usage.md#tach-sync)
+to automatically configure dependency rules.
```

Comment:
It might be nice to have a little table in the format

Command: action

I have an outstanding task to do another pass on the docs, so def nonblocking